README.md

```diff
@@ -22,29 +22,24 @@ be downloaded from
 
 To operate the production Jenkins (or more generally to access the
 production namespace), you must have access to the cluster
-at https://console.apps.ci.centos.org:8443 and to the projects
-"fedora-coreos" and "fedora-coreos-devel".
+at https://console-openshift-console.apps.ocp.ci.centos.org/
+and to the "fedora-coreos" project.
 
 For more information on CentOS CI, see these resources:
 
   - https://github.com/centosci/ocp4-docs (the "new" OCP 4.4 cluster)
   - https://wiki.centos.org/QaWiki/CI (out of date)
 
 If you need access, you can open a bug request at
-https://bugs.centos.org/ against the `Buildsys` project and
-`Ci.centos.org Ecosystem Testing` category.
-Do note that the bug-tracker and OpenShift use different
-authentication backends, so you will have separate credentials
-for each.
-
-Please specify the requested username, RBAC for projects
-`fedora-coreos` and `fedora-coreos-devel`, and your GPG key
-fingerprint.
+https://pagure.io/centos-infra with your
+[ACO credentials](https://accounts.centos.org/) similar to
+[this one](https://pagure.io/centos-infra/issue/144).
```

Comment:
This link is 404ing for me.  Is the bug restricted?

Reply:
404 for me too, not sure why

Reply:
https://github.com/coreos/fedora-coreos-pipeline/pull/305